adding example sections for DBaaS API (#881)
percona/pmm: markdown/dbaas.md

          "1.10.0": {
            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-1-logcollector",
            "image_hash": "8f106b1e9134812b77f4e210ad0fcd7d8d3515a90fe53554d24cd49defc9e044",
            "status": "available"
          },
          "1.10.0-2": {
            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-logcollector-8.0.25",
            "image_hash": "d69dad98900532e2ad6d0bf12c34a148462816fa3ee4697e9b73efef7583901a",
            "status": "recommended",
            "default": true
          }
        }
      }
    }
  ]
}
```

From this response, choose one of the images in the `pxc` section:
```json
      "matrix": {
        "pxc": {
          "8.0.19-10.1": {
            "image_path": "percona/percona-xtradb-cluster:8.0.19-10.1",
            "image_hash": "1058ae8eded735ebdf664807aad7187942fc9a1170b3fd0369574cb61206b63a",
            "status": "available"
          },
          "8.0.20-11.1": {
            "image_path": "percona/percona-xtradb-cluster:8.0.20-11.1",
            "image_hash": "54b1b2f5153b78b05d651034d4603a13e685cbb9b45bfa09a39864fa3f169349",
            "status": "available"
          },
```

> The chosen `image_path` value is the value you should provide in the next API call as the `image` field.

Reviewer: ```suggestion
The chosen `image_path` value is the value you should provide in the next API call as the `image` field. We recommend using the one with `"status": "recommended"`
```

Then to enable DBaaS send request to `Settings/Change` endpoint like below where `IP` is public IP address or DNS name of PMM Server instance.
```bash
curl -X POST "http://localhost/v1/Settings/Change" -H "accept: application/json" -u "admin:admin" -H "Content-Type: application/json" -d "{ \"pmm_public_address\": \"${IP}\", \"enable_dbaas\": true}"
```

API endpoint used in this step: [Change settings](https://percona-pmm.readme.io/reference/changesettings).

### Registering new Kubernetes cluster

Once kubernetes cluster is created it should be registered in PMM where `my_cluster` is a name of kubernetes cluster which will be used later. `sed` command is used to remove newlines, otherwise this script doesn’t work.
```bash
KUBECONFIG=$(kubectl -- config view --flatten --minify | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/\\n/g')

curl -X POST "http://localhost/v1/management/DBaaS/Kubernetes/Register" -H "accept: application/json" -u "admin:admin" -d "{ \"kubernetes_cluster_name\": \"my_cluster\", \"kube_auth\": { \"kubeconfig\": \"${KUBECONFIG}\" }}"
```
This command will register kubernetes cluster, start monitoring of kubernetes cluster and install required kubernetes operators.

> API endpoint used in this step: [RegisterKubernetesCluster](https://percona-pmm.readme.io/reference/registerkubernetescluster)

Reviewer 2: ```suggestion
API endpoint used in this step: [RegisterKubernetesCluster](ref:registerkubernetescluster)
```

```

API endpoint used in this step: [Change settings](https://percona-pmm.readme.io/reference/changesettings).

### Registering new Kubernetes cluster

Once kubernetes cluster is created it should be registered in PMM where `my_cluster` is a name of kubernetes cluster which will be used later. `sed` command is used to remove newlines, otherwise this script doesn’t work.
```bash
KUBECONFIG=$(kubectl -- config view --flatten --minify | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/\\n/g')

curl -X POST "http://localhost/v1/management/DBaaS/Kubernetes/Register" -H "accept: application/json" -u "admin:admin" -d "{ \"kubernetes_cluster_name\": \"my_cluster\", \"kube_auth\": { \"kubeconfig\": \"${KUBECONFIG}\" }}"
```
This command will register kubernetes cluster, start monitoring of kubernetes cluster and install required kubernetes operators.

API endpoint used in this step: [RegisterKubernetesCluster](https://percona-pmm.readme.io/reference/registerkubernetescluster)

### Create PXC Cluster

Once we registered kubernetes cluster we can use it’s name to create DB Clusters. Here is an example for PXC Cluster.

> 

Reviewer: should we mention that values used here is a recommended ones?

Reviewer 3: Yes

- https://percona-pmm.readme.io/reference/registerkubernetescluster
- https://percona-pmm.readme.io/reference/createpxccluster
- https://percona-pmm.readme.io/reference/createpsmdbcluster

> In this example we would use minikube for the kubernetes cluster and create PXC DB cluster, but similar API endpoints exist for the PSMDB.

Reviewer 3: ```suggestion
In this example we would use minikube for the kubernetes cluster and will create a PXC DB cluster, but similar API endpoints exist for the PSMDB.
```

```
This command will register kubernetes cluster, start monitoring of kubernetes cluster and install required kubernetes operators.

API endpoint used in this step: [RegisterKubernetesCluster](https://percona-pmm.readme.io/reference/registerkubernetescluster)

### Create PXC Cluster

Once we registered kubernetes cluster we can use it’s name to create DB Clusters. Here is an example for PXC Cluster.

```bash
curl -X POST "http://localhost/v1/management/DBaaS/PXCCluster/Create" -H "accept: application/json" -u “admin:admin” -H "Content-Type: application/json" -d "{ \"kubernetes_cluster_name\": \"my_cluster\", \"name\": \"my-cluster-1\", \"expose\": true, \"params\": { \"cluster_size\": 3, \"pxc\": { \"compute_resources\": { \"cpu_m\": 1000, \"memory_bytes\": 2000000000 }, \"disk_size\": 25000000000, \"image\": \"percona/percona-xtradb-cluster:8.0.25-15.1\" }, \"haproxy\": { \"compute_resources\": { \"cpu_m\": 1000, \"memory_bytes\": 2000000000 } } }}"
```

API endpoint used in this step: [CreatePXCCluster creates](https://percona-pmm.readme.io/reference/createpxccluster).

### List Kubernetes clusters

> Once you created PXC cluster you can check the status of the cluster by calling `List` endpoint.

Reviewer 3: ```suggestion
Once you created PXC cluster you can check the status of the cluster by calling the `List` endpoint.
```
